jmix-flowui/flowui/src/main/java/io/jmix/flowui/xml/layout/loader/component/AbstractGridLoader.java

```diff
@@ -256,6 +258,11 @@ protected void loadEditorActionsColumn(T resultComponent, Element columnElement)
         editColumn.setEditorComponent(actions);
 
         loadString(columnElement, "key", editColumn::setKey);
+        if (Strings.isNullOrEmpty(editColumn.getKey())) {
```

Comment:
This doesn't fix existing settings, so anyone upgrading to the new version will still run into this issue